Irina/src/main/java/oop/Order.java

```diff
@@ -0,0 +1,15 @@
+package oop;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public interface Order {
+
+    int sum();
+
+    int count();
+
+    LocalDateTime date();
```

Comment:
Method name should contain a verb, please rename this method!

Reply:
Done